TESTING.md

```diff
@@ -175,6 +175,12 @@ Prebuilt development images can be downloaded from here:
 
 Install the package `qemu-system-x86_64` if you don't have already installed.
 
+### Build a mesh network
+
+Up to 10 qemu nodes can be setup. Use the `--node-number N`. All the node's LAN interfaces are
+bridged together. You can use `--enable-wan` in only one of the nodes to share your internet connection
```

Comment:
Can we have only one upstream connection?

Reply:
Not necesarly but the current code of the setup script yes has this limitation as it runs a dnsmasq listening on a fixed (so a second dnsmasq will collide) and also it uses a fixed IP.  
We can improve the script to give more than one node an upstream connection. We should check if dnsmasq is already running and use a different IP per node.